apps/contrib/templatetags/verbose_name.py

```diff
@@ -0,0 +1,9 @@
+from django import template
+
+
+register = template.Library()
+
+
+@register.filter
+def verbose_name(value):
+    return value._meta.verbose_name
```

Comment:
Just because it was there before (used in the type-property) and somehow looks better to me. But doesn't really matter, anyway.